specification/sql/resource-manager/Microsoft.Sql/preview/2018-06-01-preview/managedDatabaseSensitivityLabels.json

```diff
@@ -425,9 +499,22 @@
           "description": "The label name.",
           "type": "string"
         },
+        "labelId": {
+          "description": "The label ID.",
+          "type": "string"
+        },
         "informationType": {
           "description": "The information type.",
           "type": "string"
+        },
+        "informationTypeId": {
+          "description": "The information type ID.",
+          "type": "string"
+        },
+        "isDisabled": {
+          "description": "Applicable for recommended sensitivity label source only. Specifies whether the recommended sensitivity label is disabled or not.",
```

Comment:
can use something like this for the description:
"Is sensitivity recommendation disabled. Applicable for recommended sensitivity label source only. Specifies whether the sensitivity recommendation on this column is disabled (dismissed) or not."

Reply:
Done